align/primitive/default/canvas.py

```diff
@@ -132,3 +143,8 @@ def _find_adjoining_layers( self, layer):
                 pm = m0
         assert nm is not None or pm is not None, f"Could not trace any connections for {layer}"
         return (pm, pv, nv, nm)
+
+def single_centered_via(rect, WidthX, WidthY):
+    xpos = ( rect[0] + rect[2] ) // 2
+    ypos = ( rect[1] + rect[3] ) // 2
+    return [xpos - WidthX // 2, ypos - WidthY // 2, xpos + WidthX // 2, ypos + WidthY // 2]
```

Comment:
@parijatm Should we still be doing this in the postprocessor? I suspect that PEX is broken because of this.

Reply:
Sorry single_centered_via is no longer being used. Should have removed this.